pandas/core/indexes/period.py

```diff
@@ -539,16 +543,13 @@ def asof_locs(self, where, mask):
     def astype(self, dtype, copy=True, how='start'):
         dtype = pandas_dtype(dtype)
 
-        # We have a few special-cases for `dtype`.
-        # Failing those, we fall back to astyping the values
-
         if is_datetime64_any_dtype(dtype):
             # 'how' is index-speicifc, isn't part of the EA interface.
             tz = getattr(dtype, 'tz', None)
             return self.to_timestamp(how=how).tz_localize(tz)
 
```

Comment:
hmm what does 'how' refer?

Reply:
for datetime64 dtypes this calls `to_timestamp`, which takes how to specify start/end

Reply:
ok, typo in index-speicific

Reply:
typo